fmriprep/cli/parser.py

```diff
@@ -174,6 +174,10 @@ def _bids_filter(value):
     g_conf.add_argument(
         '--dummy-scans', required=False, action='store', default=None, type=int,
         help='Number of non steady state volumes.')
+    g_conf.add_argument(
+        '--random-seed', action='store', type=int, default=None,
+        help='Initialize the random seed for the workflow'
+    )
```

Comment:
yep, let's punt reuse to a separate PR, since we'll be pulling more than random seeds.

if we want to preserve the current output structure, my workflow for reuse was essentially:
- check if `fmriprep/sub-N/log` exists
- use the latest mtime of potential uuid folders
- load up toml

I still think we should be explicit when reusing a previous config file, whether its in the workdir or outputs.